torch/ao/quantization/_quantize_dbr.py

```diff
@@ -123,7 +137,7 @@ def convert(model: torch.nn.Module) -> torch.nn.Module:
     static_mappings = get_default_static_quant_module_mappings()
     dynamic_mappings = get_default_dynamic_quant_module_mappings()
     # swap the modules
-    _swap_child_modules(model, static_mappings, dynamic_mappings)
+    _swap_child_modules(model, static_mappings, dynamic_mappings, None)
```

Comment:
nit: can we add a default None for the last argument instead of passing None around?